var/spack/repos/builtin/packages/root/package.py

```diff
@@ -283,7 +283,7 @@ class Root(CMakePackage):
     depends_on("pkgconfig", type="build")
 
     # 6.32.00 requires sys/random.h
-    depends_on("libc", when="@6.32.00:")
+    depends_on("libc", when="@6.32.00: platform=linux")
```

Comment:
```suggestion
```

Reply:
Doesn't this now require moving the version range to be added to the glibc and musl lines below?

Reply:
Yeah, the dependency exists unconditionally